src/build.d

```diff
@@ -582,9 +582,11 @@ auto sourceFiles()
     {
         assert(0, "Unknown TARGET_CPU: " ~ env["TARGET_CPU"]);
     }
+    string[] packages = [/*main src dir*/"","lib"];
     Sources sources = {
         frontend:
-            dirEntries(env["D"], "*.d", SpanMode.shallow)
+            packages.map!(p => env["D"].buildPath(p).dirEntries("*.d", SpanMode.shallow))
```

Comment:
Would just `"{,lib}*.d" work?
See also: https://dlang.org/library/std/path/glob_match.html

Reply:
I'm not sure.